.github/labels.yml

```
@@ -0,0 +1,277 @@
+# ##### pull-request-size bot labels ##### #
+- color: "009900" # green
+  description: PR that changes 0-9 lines. Quick fix/merge.
+  name: size/XS
+- color: 77bb00 # yellow-green
+  description: PR that changes 10-29 lines. Very easy to review.
+  name: size/S
+- color: eebb00 # yellow
+  description: PR that changes 30-99 lines. Good size to review.
+  name: size/M
+- color: ee9900 # yellow-orange
+  description: PR that changes 100-499 lines. Requires some effort to review.
+  name: size/L
+- color: ee5500 # orange
+  description: PR that changes 500-999 lines. Consider splitting work into several ones that easier to review.
+  name: size/XL
+- color: ee0000 # red
+  description: PR that changes 1000+ lines. You should absolutely split your PR into several.
+  name: size/XXL
+
+# ##### complexity labels ##### #
+- color: bfe5bf # green
+  description: ""
+  name: complexity:easy
+- color: F0DA27 # yellow
+  description: ""
+  name: complexity:medium
+- color: fad8c7 # red
+  description: ""
+  name: complexity:hard
+
+# ##### TSC labels ##### #
+- color: f58220
+  description: PR managing GOVERNANCE or other TSC documentation
+  name: TSC
+- color: f58220
+  description: Items related to Technical Steering Committee voting
+  name: TSC:voting
+
+# ##### type labels ##### #
+- color: b60205
+  description: ""
+  name: breaking change
+- color: d73a4a # red (standard github)
+  description: Something isn't working
+  name: bug
+- color: d73a4a
+  description: ""
+  name: bug fix
```

Let's replace `bug fix` with just `bug`. If it's an issue labeled `bug`, then it describes a bug. If it's a PR labeled `bug` then it fixes a bug.
```suggestion
- color: d73a4a # red (standard github)
  description: Something isn't working
  name: bug
```